### PR TITLE
switch to the new YAML structures and versions

### PR DESCRIPTION
**Pull request by jstrachan**
lets use separate versions for team, che, jenkins so that each can be tested and released independently
lets also allow a users tenant to be configured with a specific team/che/jenkins version and/or maven repo so that we can use acceptance tests in prod for PRs

### CODE REVIEW

**Comment by fabric8cd** [comment]
@jstrachan snapshot fabric8-tenant image is available for testing.  `docker pull docker.io/fabric8/fabric8-tenant:SNAPSHOT-PR-294-1`

**Comment by fabric8cd** [comment]
@jstrachan snapshot fabric8-tenant image is available for testing.  `docker pull docker.io/fabric8/fabric8-tenant:SNAPSHOT-PR-294-2`

**Comment by fabric8cd** [comment]
@jstrachan snapshot fabric8-tenant image is available for testing.  `docker pull docker.io/fabric8/fabric8-tenant:SNAPSHOT-PR-294-3`

**Comment by fabric8cd** [comment]
@jstrachan snapshot fabric8-tenant image is available for testing.  `docker pull docker.io/fabric8/fabric8-tenant:SNAPSHOT-PR-294-5`

File: controller/tenant.go [hunk 1]
```diff
@@ -27,16 +30,22 @@ type TenantController struct {
 	keycloakConfig  keycloak.Config
 	openshiftConfig openshift.Config
 	templateVars    map[string]string
+	usersURL        string
 }
 
 // NewTenantController creates a status controller.
 func NewTenantController(service *goa.Service, tenantService tenant.Service, keycloakConfig keycloak.Config, openshiftConfig openshift.Config, templateVars map[string]string) *TenantController {
+	usersURL := os.Getenv("F8_WIT_URL")
```
Comment (aslakknutsen) [inline controller/tenant.go]: Move to configuration.go

Comment (jstrachan) [reply controller/tenant.go]: will do!

**Comment by fabric8cd** [comment]
@jstrachan snapshot fabric8-tenant image is available for testing.  `docker pull docker.io/fabric8/fabric8-tenant:SNAPSHOT-PR-294-7`

**Comment by fabric8cd** [comment]
@jstrachan snapshot fabric8-tenant image is available for testing.  `docker pull docker.io/fabric8/fabric8-tenant:SNAPSHOT-PR-294-8`

File: controller/tenant.go (hunk 1 above)
Comment (jstrachan) [reply controller/tenant.go]: done!

**Comment by fabric8cd** [comment]
@jstrachan snapshot fabric8-tenant image is available for testing.  `docker pull docker.io/fabric8/fabric8-tenant:SNAPSHOT-PR-294-9`

**Pull request merged**